lib/components/Button.tsx

```diff
@@ -11,22 +13,26 @@ const Button: FC<PropsWithChildren<Props>> = ({
   form,
   children,
   disabled = false,
+  hidden,
+  className,
   variant = 'solid',
   onClick,
 }) => {
   const outlinedButton =
-    'h-[54px] w-[250px] md:w-32 text-ocf-yellow border-ocf-yellow border-[2px] rounded-md font-semibold hover:bg-ocf-yellow hover:text-black transition-all duration-300';
+    'h-[54px] w-[200px] text-xl text-ocf-yellow border-ocf-yellow border-[2px] rounded-md font-semibold hover:bg-ocf-yellow hover:text-black transition-all duration-300';
 
   const solidButton =
-    'inline-flex gap-[10px] items-center justify-center md:w-32 w-10/12 bg-ocf-yellow text-black disabled:bg-ocf-gray disabled:text-ocf-black-600 transition-all duration-300 shadow h-14 max-w-sm text-center rounded-md md:rounded-lg md:font-semibold font-bold text-[16px]';
+    'inline-flex gap-[10px] items-center justify-center md:w-[200px] bg-ocf-yellow text-black disabled:bg-ocf-gray disabled:text-ocf-black-600 transition-all duration-300 shadow h-14 max-w-sm text-center rounded-md md:rounded-lg md:font-semibold font-bold text-xl';
 
   return (
     <button
       suppressHydrationWarning
       form={form}
       onClick={onClick}
       disabled={disabled}
-      className={variant === 'solid' ? solidButton : outlinedButton}
+      className={`${variant === 'solid' ? solidButton : outlinedButton} ${
+        hidden && `${hidden}:hidden`
+      } ${className && className}`}
```

Comment:
You could default this variable to `''` in the parameters if it doesn't type check with just below
```suggestion
      } ${className}`}
```